docs/internals/layout_in_storage.rst

```diff
@@ -246,168 +250,211 @@ value and reference types, types that are encoded packed, and nested types.
-          "label": "b1",
-          "offset": 0,
-          "slot": "10",
-          "type": "t_bytes_storage"
-        }
-      ],
-      "types": {
-        "t_address": {
-          "encoding": "inplace",
-          "label": "address",
-          "numberOfBytes": "20"
-        },
-        "t_array(t_uint256)2_storage": {
-          "base": "t_uint256",
-          "encoding": "inplace",
-          "label": "uint256[2]",
-          "numberOfBytes": "64"
-        },
-        "t_array(t_uint256)dyn_storage": {
-          "base": "t_uint256",
-          "encoding": "dynamic_array",
-          "label": "uint256[]",
-          "numberOfBytes": "32"
-        },
-        "t_bool": {
-          "encoding": "inplace",
-          "label": "bool",
-          "numberOfBytes": "1"
-        },
-        "t_bytes_storage": {
-          "encoding": "bytes",
-          "label": "bytes",
-          "numberOfBytes": "32"
-        },
-        "t_mapping(t_address,t_bool)": {
-          "encoding": "mapping",
-          "key": "t_address",
-          "label": "mapping(address => bool)",
-          "numberOfBytes": "32",
-          "value": "t_bool"
-        },
-        "t_mapping(t_uint256,t_mapping(t_address,t_bool))": {
-          "encoding": "mapping",
-          "key": "t_uint256",
-          "label": "mapping(uint256 => mapping(address => bool))",
-          "numberOfBytes": "32",
-          "value": "t_mapping(t_address,t_bool)"
-        },
-        "t_string_storage": {
-          "encoding": "bytes",
-          "label": "string",
-          "numberOfBytes": "32"
-        },
-        "t_struct(S)13_storage": {
-          "encoding": "inplace",
-          "label": "struct A.S",
-          "members": [
-            {
-              "astId": 3,
-              "contract": "fileA:A",
-              "label": "a",
-              "offset": 0,
-              "slot": "0",
-              "type": "t_uint128"
-            },
-            {
-              "astId": 5,
-              "contract": "fileA:A",
-              "label": "b",
-              "offset": 16,
-              "slot": "0",
-              "type": "t_uint128"
-            },
-            {
-              "astId": 9,
-              "contract": "fileA:A",
-              "label": "staticArray",
-              "offset": 0,
-              "slot": "1",
-              "type": "t_array(t_uint256)2_storage"
-            },
-            {
-              "astId": 12,
-              "contract": "fileA:A",
-              "label": "dynArray",
-              "offset": 0,
-              "slot": "3",
-              "type": "t_array(t_uint256)dyn_storage"
-            }
-          ],
-          "numberOfBytes": "128"
-        },
-        "t_uint128": {
-          "encoding": "inplace",
-          "label": "uint128",
-          "numberOfBytes": "16"
-        },
-        "t_uint256": {
-          "encoding": "inplace",
-          "label": "uint256",
-          "numberOfBytes": "32"
-        }
+  {
```

Comment:
Can you keep the original indent so that the diff does not show it as if it was all replaced? Our style is 4 spaces so no reason to change it anyway.

```suggestion
    {
```